test/Test_Array.re

```diff
@@ -142,11 +121,34 @@ describe("Array", () => {
   });
 
   describe("Unfoldable", () => {
-    open Array.Unfoldable;
-    it("should do unfold", () => {
-      expect(unfold(x => if (x>5) None else Some((x, x+1)), 0)) |> to_be([|0, 1, 2, 3, 4, 5|]);
-      expect(unfold(x => if (x>20) None else Some((x, x+5)), 0)) |> to_be([|0, 5, 10, 15, 20|]);
-    });
+    Array.Unfoldable.(
```

Comment:
This is `refmt` doing this right? I say just leave the old `open`  in there since it's already a local open anyway so it's not any better, just a lot harder to read

Reply:
Yes it's the formatter :confused:

I can revert these style changes now if needed :+1: